lib/client-options.js

```diff
@@ -231,14 +231,12 @@ const errors = require("./errors.js");
  * true allows you to use ECMAScript undefined as Cassandra `unset`.
  *
  * Default: true.
- * @property {Boolean} [encoding.useBigIntAsLong] Use [BigInt ECMAScript type](https://tc39.github.io/proposal-bigint/)
- * to represent CQL bigint and counter data types.
- * [TODO: Add support for this field]
- * @property {Boolean} [encoding.useBigIntAsVarint] Use [BigInt ECMAScript type](https://tc39.github.io/proposal-bigint/)
- * to represent CQL varint data type.
+ * @property {Boolean} [encoding.useBigIntAsLong] Use [BigInt type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt)
+ * to represent CQL bigint and counter data types. Defaults to true.
+ * @property {Boolean} [encoding.useBigIntAsVarint] Use [BigInt type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt)
+ * to represent CQL varint data type. Defaults to true.
  *
  * Note, that using Integer as Varint (`useBigIntAsVarint == false`) is deprecated.
```

Comment:
>  * Note, that using Integer as Varint (`useBigIntAsVarint == false`) is deprecated.

:thought_balloon: This note was there already in DSx driver, right? Can you check for how long this option has been deprecated? Maybe we can consider dropping support for Integer altogether? This depends on how much BigInt has become the standard in Node.js and how established it's been.